File: backend-dotnet/EventModelingGitHubCloneDotNet/EventModelingGitHubCloneDotNet/Slices/Read/PullRequestComments/Presentation/GetPullRequestCommentsResponseBody.cs
using System.Collections.Immutable;
using EventModelingGitHubCloneDotNet.Slices.Read.PullRequestComments.Core.Application;

namespace EventModelingGitHubCloneDotNet.Slices.Read.PullRequestComments.Presentation
{
    public class GetPullRequestCommentsResponseBody
    {
        public ImmutableList<PullRequestComment> Comments { get; set; }
    }
}

File: backend-kotlin/src/main/kotlin/pl/zycienakodach/eventmodeling/githubclone/slices/read/pullrequestcomments/presentation/GetPullRequestCommentsResponseBody.kt
package pl.zycienakodach.eventmodeling.githubclone.slices.read.pullrequestcomments.presentation

import pl.zycienakodach.eventmodeling.githubclone.slices.read.pullrequestcomments.application.PullRequestComment

data class GetPullRequestCommentsResponseBody(val comments: List<PullRequestComment>)
